app/lib/features/comments/widgets/comment.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:acter/common/widgets/render_html.dart';
```

Comment:
File name suggestion:
Lets try to make all file name same as class name.

So in this case, it should be `comment_widget.dart` instead of` comment.dart`